src/types/drupal/paragraph.ts

```diff
@@ -212,13 +212,37 @@ export interface ParagraphRichTextCharLimit1000 extends DrupalParagraph {
 }
 
 export interface ParagraphServiceLocation extends DrupalParagraph {
```

Comment:
We should probably rename this...I'm guessing? I don't actually know what all uses it yet, though, so it may not be VAMC-specific. 🤷

Reply:
According to Drupal these all use it

VAMC Facility Health Service, VAMC Facility Non-clinical Service, Service Region, VBA Facility Service